app_spec/test/index.js

```diff
@@ -57,6 +57,21 @@ if (process.env.APP_SPEC_NETWORK_TYPE === "sim1h")
   );
 }
 
+if (process.env.APP_SPEC_NETWORK_TYPE === "sim2h")
+{
+    transport_config = {
+        type: 'sim2h',
+        sim2h_url: "http://localhost:9000",
```

Comment:
how is it that using `http` works here when the websocket is running at `wss://...`

Reply:
heh!  good question.  I guess the protocol type is being ignored by the our transport code.

Reply:
May as well switch it here, for correctness